TimberWinR/Outputs/Redis.cs

```diff
@@ -183,6 +201,18 @@ private bool ApplyFilters(JObject json)
             return drop;
         }
 
+
+        // 
+        // What is average queue depth?
+        //
+        private int AverageQueueDepth()
+        {
+            lock(_locker)
+            {
+                return (int)_sampleQueueDepths.Average();
```

Comment:
Forgive my C# lack of knowledge, but is there a risk of overflow since the _sampleQueueDepths is a long array and the the return value is an int?